modules/exploits/linux/upnp/belkin_wemo_upnp_exec.rb

```diff
@@ -82,16 +83,38 @@ def check
     )
 
     if res && res.code == 200 && res.body.include?('urn:Belkin:device:')
-      vprint_good('Wemo-enabled device detected')
-      return CheckCode::Appears
+      print_good('Wemo-enabled device detected')
+    else
+      print_error('This does not appear to be a wemo-enabled device')
```

Comment:
Can you make these print messages consistent? Consistent style is important to me. I also capitalized Wemo.

> Wemo-enabled device not detected